### PR TITLE
ERT suit heat fix

### LINKED ISSUE #3185: ERT suits aren't fireproof
shits broken(again) yo

### PR DESCRIPTION
**Pull request by Boxe**
makes the ERT suit actually resistant to heat

also makes ERT suits bulky items because they weren't before for some fucking reason

also removes a useless list that was overwritten by another one(i think)

fixes  #3185

### CODE REVIEW

File: code/modules/clothing/spacesuits/ert.dm [hunk 1]
```diff
@@ -31,8 +32,7 @@
 	desc = "A suit worn by members of the NanoTrasen Emergency Response Team. Armoured, space ready and fire resistant."
 	icon_state = "ert_commander"
 	item_state = "suit-command"
-	w_class = 3
-	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_storage,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/melee/energy/sword,/obj/item/weapon/handcuffs,/obj/item/weapon/tank/emergency_oxygen,/obj/item/weapon/tank/emergency_nitrogen)
```
Comment (ComicIronic) [inline code/modules/clothing/spacesuits/ert.dm]: Why did you remove this?

Comment (Boxe) [inline code/modules/clothing/spacesuits/ert.dm]: i think it's overwritten by the allowed = list( on line 40, could you confirm?

Comment (ComicIronic) [inline code/modules/clothing/spacesuits/ert.dm]: So it is.
Odd that the compiler didn't pick up the double define.

Comment (d3athrow) [inline code/modules/clothing/spacesuits/ert.dm]: because its not a definition.
